### PR TITLE
changed RAIDStatus to RaidStatus with unit test-cases

### PR DESCRIPTION
**Pull request by Rachit7194**
Hi Chris,

Before adding this change to upstream dracclient, i would like to review dracclient patch by you first.
I have added 1 test case for RAIDStatus to RaidStatus fix. 

As change is in ``_parse_drac_virtual_disk`` , i think it is private method and there is no test cases for private methods. so i added test case for ``list_virtual_disks()`` which internally calls ``_parse_drac_virtual_disk``

Please let me know if any change is need to add/update.

Thanks

### CODE REVIEW

File: dracclient/tests/utils.py [hunk 1]
```diff
@@ -246,9 +246,11 @@ def load_wsman_xml(name):
         'ok': load_wsman_xml('physical_disk_view-enum-ok')
     },
     uris.DCIM_VirtualDiskView: {
+        'Raid_Status_ok': load_wsman_xml(
+            'virtual_disk_view-enum-with-raid-status-ok'),
         'ok': load_wsman_xml('virtual_disk_view-enum-ok')
     }
-}
+   }
```
Comment (cdearborn) [inline dracclient/tests/utils.py:253]: Can you remove the leading spaces added to the above line?

**Review by cdearborn — APPROVED** [review]
Great job on this Rachit!  Just saw some leading spaces on 1 line that were accidentally added.  Please go ahead and submit the patch upstream.

File: dracclient/tests/utils.py (hunk 1 above)
Comment (Rachit7194) [reply dracclient/tests/utils.py:253]: i will do the change. thanks

**Comment by Rachit7194** [comment]
Thanks Chris..!..
I have created a patch  : https://review.opendev.org/#/c/704731/

**Comment by cdearborn** [comment]
Closing this patch, as it has been submitted upstream.

**Pull request closed without merging**